src/test/api.test.js

```diff
@@ -0,0 +1,13 @@
+import { postMovies, getMovies } from '../../_mocks_/apiMock.js';
+
+describe('countLikes to be a number', () => {
+  test('should retrieve data from API', async () => {
+    const data = await getMovies();
+    expect(data.title).toEqual('quis ut nam facilis et officia qui');
+  });
+
+  test('should send Data to API', async () => {
+    // const post = await postMovies(2, 'How are you', false);
+    expect(postMovies()).toBeDefined();
+  });
```

Comment:
- [ ] two tests are failing please make sure to fix them so that they pass without a problem 
<img width="905" alt="Screenshot 2023-03-16 at 18 59 15" src="https://user-images.githubusercontent.com/61790587/225710994-4b437962-f13d-4d59-9c38-70282b6ac9b5.png">